server/routes/__tests__/money.spec.js

```diff
@@ -338,14 +347,14 @@ describe('Prisoner Money', () => {
       const spy = jest.spyOn(prisonerInformationService, 'getTransactionsFor');
 
       await request(app)
-        .get('/money/transactions/spends?selectedDate=2022-12-01')
+        .get('/money/transactions/spends?selectedDate=2021-01-01')
```

Comment:
Reverted changes from previous breakages as didn't actually fix the problem, all other date changes to 2022 are the same below.